### DISCUSSION TITLE
Myloader Error

### DISCUSSION THREAD

**Discussion by sebleje76** [mydumper/mydumper#1449] [General]
Hi
When I restore a schema via myloader I have a problem restoring to a table with the error below:
Thread 19 : Error restoring 1 : binary ;
 "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'binary' at line 1".
Do you have any idea of the root cause?

Best regard

**Comment by davidducos** [comment]
are you exporting and importing on the same version of database?

**Comment by sebleje76** [comment]
The source version is 10.3 and the target version is 10.5.

I have an import problem just on a table following a table creation which poses a problem.

**Comment by sebleje76** [comment]
Following the upgrade of mydumper, I no longer have the problem of restoring the table.

**Comment by davidducos** [comment]
Please, use a newer version and you should use the same version when dump and import.
